src/cloud-providers/ibmcloud/provider.go

```diff
@@ -335,7 +335,7 @@ func (p *ibmcloudVPCProvider) updateInstanceProfileSpecList() error {
 }
 
 // Add a method to retrieve cpu, memory, and storage from the profile name
```

Comment:
nit the comment will need to be changed
`// Add a method to retrieve cpu, memory and arch from the profile name`